ubuntu-advantage

```diff
@@ -13,15 +25,53 @@ def get_args(argv):
     action_enable = subparsers.add_parser(
         'enable-esm', help='enable the ESM repository')
     action_enable.add_argument(
-        'token', help='the user token, in the form "<chars>:<chars>"')
+        'token', help='the user token, in the form "<user>:<password>"')
 
     subparsers.add_parser('disable-esm', help='disable the ESM repository')
 
     return parser.parse_args(argv)
 
 
+def enable_esm(token, lists_dir=None, print=print):
+    """Enable ESM archive with the specified token."""
+    list_file = get_list_file(lists_dir=lists_dir)
+    write_list_file(list_file, token)
+    print(
+        'Ubuntu ESM repository enabled.'
+        ' Run "sudo apt-get update" to update lists.')
+
+
+def disable_esm(lists_dir=None, print=print):
+    """Disable ESM archive."""
+    list_file = get_list_file(lists_dir=lists_dir)
+    if os.path.exists(list_file):
+        list_file_disabled = list_file + '.save'
+        os.rename(list_file, list_file_disabled)
+        print('Ubuntu ESM repository disabled.')
```

Comment:
Who is the consumer of this message? Do we not need to add 'Run "sudo apt-get update" to update lists.' here as well to avoid installing packages already in apt cache?